app/providers/atmosphere/optimization_strategy/default.rb

```diff
@@ -105,7 +120,7 @@ def select
           best_tenant = nil
           best_flavor = nil
           instantiation_cost = Float::INFINITY
-          tmpls.each do |tmpl|
+          @tmpls.each do |tmpl|
```

Comment:
Uh, I don't believe I understand. `@tmpls` is an instance variable which I'm iterating over. `tmpl` is a local variable which is only needed in the context of this specific loop - hence no `@`. What good would replacing `tmpl` with `@tmpl` do? Seems like a pointless waste of RAM.